modules/monitoring-configuring-remote-write-storage.adoc

```diff
@@ -126,14 +212,14 @@ data:
           ca:
             secret:
               name: selfsigned-mtls-bundle
-              key: ca.crt
+              key: <ca.crt>
           cert:
             secret:
               name: selfsigned-mtls-bundle
-              key: client.crt
+              key: <client.crt>
           keySecret:
             name: selfsigned-mtls-bundle
-            key: client.key
+            key: <client.key>
```

Comment:
see from https://deploy-preview-44713--osdocs.netlify.app/openshift-enterprise/latest/monitoring/configuring-the-monitoring-stack.html#configuring_remote_write_storage_configuring-the-monitoring-stack
the `tlsConfig` configmap format is different with others

Reply:
fixed